frontend/packages/dev-console/src/components/import/app/ApplicationSelector.tsx

```diff
@@ -30,29 +32,39 @@ const ApplicationSelector: React.FC<ApplicationSelectorProps> = ({ namespace })
     validateForm();
   };
 
+  const handleOnLoad = (items: { [key: string]: string }) => {
+    setIsZeroApplication(_.isEmpty(items));
```

Comment:
Should store the result of `_.isEmpty(items)` in a local var to reference on the next line.